dspace-api/src/main/java/org/dspace/access/status/DefaultAccessStatusHelper.java

```diff
@@ -156,4 +162,83 @@ private String caculateAccessStatusForDso(Context context, DSpaceObject dso, Dat
         }
         return RESTRICTED;
     }
+
+    /**
+     * Look at the policies of the primary (or first) bitstream of the item to retrieve its embargo.
+     *
+     * If the item is null, simply returns an empty map with no embargo information.
+     *
+     * @param context     the DSpace context
+     * @param item        the item to embargo
+     * @return an access status value
+     */
+    @Override
+    public String getEmbargoFromItem(Context context, Item item)
+            throws SQLException {
+        Date embargoDate;
+
+        if (item == null) {
+            return null;
+        }
+        // Consider only the original bundles.
+        List<Bundle> bundles = item.getBundles(Constants.DEFAULT_BUNDLE_NAME);
+        // Check for primary bitstreams first.
+        Bitstream bitstream = bundles.stream()
+                .map(bundle -> bundle.getPrimaryBitstream())
+                .filter(Objects::nonNull)
+                .findFirst()
+                .orElse(null);
+        if (bitstream == null) {
+            // If there is no primary bitstream,
+            // take the first bitstream in the bundles.
+            bitstream = bundles.stream()
+                    .map(bundle -> bundle.getBitstreams())
+                    .flatMap(List::stream)
+                    .findFirst()
+                    .orElse(null);
+        }
+
+        if (bitstream == null) {
+            return null;
+        }
+
+        embargoDate = this.retrieveLongestEmbargo(context, bitstream);
+
+        return embargoDate != null ? embargoDate.toString() : null;
+    }
+
+    /**
+     *
+     */
+    private Date retrieveLongestEmbargo(Context context, Bitstream bitstream) throws SQLException {
+        Date embargoDate = null;
+        // Only consider read policies.
+        List<ResourcePolicy> policies = authorizeService
+                .getPoliciesActionFilter(context, bitstream, Constants.READ);
+
+        // Looks at all read policies.
+        for (ResourcePolicy policy : policies) {
+            boolean isValid = resourcePolicyService.isDateValid(policy);
+            Group group = policy.getGroup();
+
+            if (group != null && StringUtils.equals(group.getName(), Group.ANONYMOUS)) {
+                // Only calculate the status for the anonymous group.
+                if (!isValid) {
+                    // If the policy is not valid there is an active embargo
+                    Date startDate = policy.getStartDate();
```

Comment:
I think this assumption is necessary because it's how the general idea of an embargo is defined in DSpace: https://wiki.lyrasis.org/display/DSDOC7x/DSpace+Item+State+Definitions

So, the behavior of this PR is correct.  An embargo always has a start date.   

As @paulo-graca points out though, it's possible to create an "access condition" named `embargo` that *doesn't have a start date*.  In that situation though the name `embargo` is just a name... it doesn't change the behavior of embargoes in DSpace's UI/REST API.  

In the UI/REST API, an Item/Bitstream will appear embargoed whenever it has a policy which has a `start date` in the future.  That's hardcoded into how DSpace works, and isn't actually configurable.  

(The `access-condition.xml` is only used right now in the Submission / Batch Editing UIs to provide an easier way to configure access settings...but those configs don't impact how DSpace behaves with respect to embargoes or leases.)